src/platform/macos/display.mm

```diff
@@ -135,7 +135,7 @@
 
   std::shared_ptr<display_t>
   display(platf::mem_type_e hwdevice_type, const std::string &display_name, const video::config_t &config) {
-    if (hwdevice_type != platf::mem_type_e::system) {
+    if (hwdevice_type != platf::mem_type_e::system && hwdevice_type != platf::mem_type_e::videotoolbox) {
```

Comment:
Do we still support `platf::mem_type_e::system`?

Reply:
Yep, the software encoder uses that.